### ISSUE TITLE
Request: texinfo (texi) files highligher for editors

### ISSUE THREAD

**Issue by estatistics** [ScintillaOrg/lexilla#74]
Request: texinfo (texi) files highligher for editors

I am a translator of pspp of texi files. However in geany, an editor, no highlight for texi files exist which uses your project for highlight. 

I was informed that your project is written is c++. 

I have the below file written in lua format. 
It may be useful for some people if u can implement it in lua project. 

[texinfo.txt](https://github.com/ScintillaOrg/lexilla/files/8441181/texinfo.txt)

**Comment by zufuliu** [comment]
https://www.gnu.org/software/texinfo/manual/texinfo/

**Comment by estatistics** [comment]
Yes, i know this page. I have found also textadept editor that provides highlight for texi files. The uploaded file is from that project
